docker-compose.yml

```diff
@@ -162,13 +160,12 @@ services:
         ipv4_address: $MANAGER_IP
   middleware:
     container_name: middleware
-    image: ghcr.io/runcitadel/middleware:v0.0.11@sha256:e472da8cbfa67d9a9dbf321334fe65cdf20a0f9b6d6bab33fdf07210f54e7002
+    image: ghcr.io/runcitadel/middleware:main@sha256:2aa20f31001ab9e61cda548acbd1864a598728731ad6121f050c6a41503866ae
     depends_on:
-    - manager
     - bitcoin
     - lightning
     - redis
-    command: sh -c "./wait-for-manager.sh $MANAGER_IP && ./start.sh"
```

Comment:
Removing this causes issues with JWT. As the auth code is written now the manager needs to be running before middleware otherwise it's not possible to log in.